lib/pbench/server/api/__init__.py

```diff
@@ -144,6 +148,17 @@ def create_app(server_config):
 
     app.logger = get_pbench_logger(__name__, server_config)
 
+    # sets up email configuration
+    mail_settings = {
+        "MAIL_SERVER": "smtp.gmail.com",
+        "MAIL_PORT": 465,
+        "MAIL_USE_TLS": False,
+        "MAIL_USE_SSL": True,
+        "MAIL_USERNAME": server_config.get("pbench-server", "email_username"),
+        "MAIL_PASSWORD": server_config.get("pbench-server", "email_password")
```

Comment:
It would be good to name the configuration option more specifically than `"email_username"`.  I can imagine that there might be several email addresses that we would want to configure.  This one is the one used to send mail _from_.  (Another might be one which the server uses to send periodic status reports _to_; another might be one for sending "alerts" to; etc....once you endow the server with the ability to send email, we may find that there are lots of things it wants to send!)